src/components/common/NavigationBar.tsx

```diff
@@ -0,0 +1,104 @@
+import styled from 'styled-components';
+import { COLOR } from '@src/styles/color';
+import { FONT_STYLES } from '@src/styles/fontStyle';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import ImageDiv from './ImageDiv';
+import { imgLogo } from 'public/assets/images';
+
+function NavigationBar() {
+  const router = useRouter();
+
+  return (
+    <StNavigationBar>
+      <ImageDiv className="logo" priority src={imgLogo} layout="fill" alt="" />
+      <nav>
+        <StTabList>
+          <StTab>
+            <Link href="/home">
+              <a className={router.pathname === '/home' ? 'clicked' : ''}>홈</a>
+            </Link>
+          </StTab>
+          <StTab>
+            <Link href="/learn">
+              <a className={router.pathname === '/learn' ? 'clicked' : ''}>학습하기</a>
+            </Link>
+          </StTab>
+          <StTab>
+            <Link href="/review">
+              <a className={router.pathname === '/review' ? 'clicked' : ''}>복습하기</a>
+            </Link>
+          </StTab>
```

Comment:
```suggestion
          <StTab isActive={router.pathname === '/home'}>
            <Link href="/home">
              <a>홈</a>
            </Link>
          </StTab>
          <StTab isActive={router.pathname === '/learn'}>
            <Link href="/learn">
              <a>학습하기</a>
            </Link>
          </StTab>
          <StTab isActive={router.pathname === '/review'}>
            <Link href="/review">
              <a>복습하기</a>
            </Link>
          </StTab>
```

className 대신 props를 활용하면 더 좋을 것 같네요! 😊

Reply:
깔끔하다!!

Reply:
좋아요!! 수정하겠습니다